src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GameStarLogo.java

```diff
@@ -8,11 +8,6 @@
 import org.openqa.selenium.Dimension;
 import org.testng.annotations.Test;
 
-/**
- * Bartosz 'V.' Bentkowski
- *
- * @ownership Content X-Wing Wikia
- */
 public class GameStarLogo extends NewTestTemplate {
 
   private final static String deWikiName = "de.gta";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)